### PR TITLE
Add new topic aboud clean code lessons podcasts by Robert C Martin

### PR DESCRIPTION
**Pull request by Fazer01**
## What does this PR do?
Add podcast/screencast on Clean Coding topic by Robert C. Martin
### Description
A brilliant screencast on the topic of Clean Coding explained by Robert C. Marin a.k.a. Uncle Bob

### Why is this valuable (or not)?
Since there was not already a (pod)(screen)cast with this subject I thought it might be valuable to add it. Because of the way of telling and point of view by Uncle Bob.

### How do we know it's really free?
It's on youtube. 

### For book lists, is it a book? For course lists, is it a course? etc.
It is a podcast / screencast. 

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: casts/free-podcasts-screencasts-en.md [hunk 1]
```diff
@@ -57,6 +58,11 @@
 * [CppCast](http://cppcast.com) - Conor Hoekstra, Jason Turner, JeanHeyd Meneide, Matt Godbolt, Rob Irving (podcast)
 
 
+### Clean Coding
+
+* [Clean Code Lessons](https://www.youtube.com/watch?v=7EmboKQH8lM&list=PLdpsE-GEhYVn_81kDPo1mwE73UgYCeMLu) - Robert C. Martin (Youtube)
```
Comment (eshellman) [inline casts/free-podcasts-screencasts-en.md:63]: For youtube, we prefer "playlist" links and the name of the producer/creator. docs/CONTRIBUTING.md#platforms-and-access-notes

File: casts/free-podcasts-screencasts-en.md [hunk 2]
```diff
@@ -3,6 +3,7 @@
 * [Android](#android)
 * [C#](#csharp)
 * [C++](#cpp)
+* [Clean Coding](#clean-code)
```
Comment (eshellman) [inline casts/free-podcasts-screencasts-en.md:6]: use a more generic title.
It's usually not helpful to add a category for a single resource. Try to find another category, or see if there are other books in our lists that would fit into your category.

**Review by eshellman — CHANGES_REQUESTED** [review]

File: casts/free-podcasts-screencasts-en.md (hunk 2 above)
Comment (Fazer01) [reply casts/free-podcasts-screencasts-en.md:6]: Okay thanks! Will try to fit it into another category. Or find more of the same to fit into clean code.

File: casts/free-podcasts-screencasts-en.md (hunk 1 above)
Comment (Fazer01) [reply casts/free-podcasts-screencasts-en.md:63]: When I open the provided link, it referst to a playlist, see image below:
image

**Comment by eshellman** [comment]
you say this is a course, but you've put it in podcasts/ screencasts. Care to explain?

**Comment by Fazer01** [comment]
Someone on this? @eshellman @davorpa Thanks in advance!

**Comment by eshellman** [comment]
is there no place we can link to that has all the lessons?

**Comment by Fazer01** [comment]
> is there no place we can link to that has all the lessons?

That was what I had in the first commit... 
I will change it back to what I had. :)

**Comment by Fazer01** [comment]
Anyone? @davorpa  @eshellman

**Comment by eshellman** [comment]
yes that's a better link. It's not so easy for reviewers to figure out - that's why we ask for explanation in the PR description.

**Pull request merged**